MIGRATION-FROM-SCK.md

```diff
@@ -0,0 +1,349 @@
+* otelcol_processor_batch_batch_send_size_sum
+* otelcol_processor_batch_timeout_trigger_send
+* otelcol_processor_dropped_log_records
+* otelcol_processor_dropped_metric_points
+* otelcol_processor_refused_log_records
+* otelcol_processor_refused_metric_points
+* otelcol_receiver_accepted_metric_points
+* otelcol_receiver_refused_metric_points
+* otelcol_scraper_errored_metric_points
+* otelcol_scraper_scraped_metric_points
+* scrape_duration_seconds
+* scrape_samples_post_metric_relabeling
+* scrape_samples_scraped
+* scrape_series_added
+* system.cpu.load_average.15m
+* system.cpu.load_average.1m
+* system.cpu.load_average.5m
+* system.cpu.time
+* system.disk.io
+* system.disk.io_time
+* system.disk.merged
+* system.disk.operation_time
+* system.disk.operations
+* system.disk.pending_operations
+* system.disk.weighted_io_time
+* system.filesystem.inodes.usage
+* system.filesystem.usage
+* system.memory.usage
+* system.network.connections
+* system.network.dropped
+* system.network.errors
+* system.network.io
+* system.network.packets
+* system.paging.faults
+* system.paging.operations
+* system.paging.usage
+* system.processes.count
+* system.processes.created
+* up
+
+### Changes in objects in Splunk OpenTelemetry Collector for Kubernetes
+
+This chart does not support the collection of Kubernetes objects. This is future functionality. You can use SCK to collect other objects.
+
+## Migration overview
+
+### Migration options/matrix
+
+The following table shows the options for migrating from SCK to Splunk OpenTelemetry Collector for Kubernetes:
+
+| Method  | Logs | Metrics | Objects |
+|---|---|---|---|
+| SCK | Yes | Yes | Yes |
+| Splunk OpenTelemetry Collector for Kubernetes | Yes | Yes | No |
+
+As shown in the table, you can acquire logs and metrics using Splunk OpenTelemetry Collector for Kubernetes. If you have objects currently deployed and need objects data, leave your current SCK objects deployment as-is (Helm command args --set="splunk-kubernetes-logging.enabled=false,splunk-kubernetes-metrics.enabled=false,splunk-kubernetes-objects.enabled=true"). This allows you to use only the SCK for objects data together with Splunk OpenTelemetry Collector for Kubernetes for logs and other telemetry data.
+
+### Checkpoint translation
+
+With migration to the Splunk OpenTelemetry Collector for Kubernetes, the underlying framework/agent being used has changed, so there needs to be a translation for checkpoint data so that the new OpenTelemetry agent can continue where Fluentd left off. All of this occurs automatically as an initContainer when you deploy the new Helm chart in your cluster for the first time. If you have properly configured the values.yaml file for the new Helm chart, it will continue working without interruption. Without proper configuration for migration, there is a possibility of either data duplication or data loss while migrating from SCK to Splunk OpenTelemetry Collector for Kubernetes.
```

Comment:
> If you have properly configured the values.yaml file for the new Helm chart, it will continue working without interruption. Without proper configuration for migration, there is a possibility of either data duplication or data loss while migrating from SCK to Splunk OpenTelemetry Collector for Kubernetes.

This sounds somewhat confusing if we don't provide any guidance on how to properly configure that. I'd call out that configuration is required only if user has a custom path for checkpoints for fluentd, otherwise it should work OOTB.